alyssaxuu/jumpskip
# Update to manifest v3 and apply some changes for compatibilization with the new version

### BEWARE Please, test it first before merge!

Unfortunately, I don't have Netflix (and chill) account and I was not able to test it during a movie.

Also, there's some minor changes:

- Apply strict equals on the comparisons and some formatting.
- Remove jquery from the background.js.
- Use a different approach for the light/dark mode detection due the new manifest restrictions (like we cannot check the window object on the background).

For now, I just only verified that the popup interactions were working right and the icon color is changed when you're opening some tab.
